eth/tracers/api_test.go

```diff
@@ -323,148 +321,6 @@ func TestTraceCall(t *testing.T) {
 	}
 }
 
-func TestOverriddenTraceCall(t *testing.T) {
```

Comment:
why do we have to delete this test?

Reply:
It just moved below to minimise the diff to upstream and the name was changed the the upstream one as well: https://github.com/celo-org/celo-blockchain/pull/2137/files/879d1ca488177e46d97452ad75906694c10997ae#diff-a63a59094cfdc02a16570494304f3253c68fc35452b395f22d7652b9c83fdb0aR433